include/CLI/Validators.hpp

```diff
@@ -750,7 +750,7 @@ class AsNumberWithUnit : public Validator {
 
         // transform function
         func_ = [mapping, opts](std::string &input) -> std::string {
-            Number num;
+            Number num{};
```

Comment:
Not sure this is necessary, but it's okay.

Reply:
I understand that this is just a cosmetic change. I just wanted to hold the tool back from moaning.

Reply:
If it's just a cosmetic change, I'd have no problem with it. I think it might zero initialize some types, which is technically wasteful, the will only be written to. But in the scope of parsing CLI code, it's totally acceptable.

Reply:
Yep. That was my take also.

Reply:
I completely understand your point of view